docs/.sphinx/_templates/404.html

```diff
@@ -0,0 +1,17 @@
+{% extends "page.html" %}
```

Comment:
Not sure if this is because I am running it locally, but when I try to access a page that does not exists (ex. http://127.0.0.1:8000/explanation/test), the `404: Not Found` page is not very pretty

![image](https://github.com/canonical/imagecraft/assets/5464641/dd63dded-49a2-4989-bfe0-f9d0ea389c27)

And when I try to run `make epub` I also get the following error, indicating something might be wrong with the 404 page:

```
Theme error:
An error happened in rendering the page 404.
Reason: TemplateNotFound('404.html')
make: *** [Makefile:78: epub] Error 2
```

Reply:
Yes, it's because it needs a real web server for the redirects to work. The epub build is a fluke. We shouldn't even offer it :)